src/app/state/staking/saga.test.ts

```diff
@@ -118,4 +120,84 @@ describe('Staking Sagas', () => {
       ])
     })
   })
+
+  describe('Fetch validators fallbacks', () => {
```

Comment:
Strange. Tests were passing with `react-scripts test`

Reply:
react-script does not configure test timeouts as far as I am away. it's probably https://jestjs.io/docs/configuration#testtimeout-number `testTimeout`  or just jest.setTimeout(xyz) in setup, but in overall does that unit test is now making a  real call to RPC ?

Reply:
Nope, all mocked :shrug:

Reply:
I have a lead: if I upgrade to jest@27 on react-scripts I get the same error

Reply:
Apparently I must useRealTimers with expectSaga, so I'll need a different way to mock timestamp

Reply:
:/ seems like it is known issue in new Jest and sinon workaround works for me with your code. There is  legacy timer option too, but I did not look at it.
https://github.com/facebook/jest/issues/10221#issuecomment-983063883

Reply:
Just extracted into a `yield* call(now)` function. That's easy to mock :)

Much better:
```
      ✓ should load validators when switching network (3 ms)
      ✓ should use fallback on mainnet (33 ms)
```